### PR TITLE
Add hyperlink to conductR in deployment doc

### PR DESCRIPTION
**Pull request by richardimaoka**
Refs #23022 
https://richard-akka-doc.firebaseapp.com/scala/additional/deploy.html

### CODE REVIEW

File: akka-docs/src/main/paradox/scala/additional/deploy.md [hunk 1]
```diff
@@ -5,7 +5,16 @@ Akka can be used in different ways:
...
- * As an application packaged and deployed using [Lightbend ConductR](http://www.lightbend.com/products/conductr).
+ 
+## Lightbend ConductR
+ 
+An Akka application can also be packaged and deployed using [Lightbend ConductR](https://conductr.lightbend.com/docs/2.2.x/DevQuickStart).
+
+ConductR is a solution for managing Lightbend Reactive Platform applications across a cluster of machines.
+It is reactive from the ground up thus enabling operations to provide the resiliency required to unleash the full benefits 
+of the Lightbend Reactive Platform in their organization.
+
```
Comment (richard-imaoka) [inline akka-docs/src/main/paradox/scala/additional/deploy.md]: Just copied and pasted the first paragraph from https://conductr.lightbend.com/docs/2.2.x/DevQuickStart

File: akka-docs/src/main/paradox/scala/additional/deploy.md [hunk 2]
```diff
@@ -5,7 +5,16 @@ Akka can be used in different ways:
...
+![visualizer_simple.png](../../images/visualizer_simple.png)
```
Comment (richard-imaoka) [inline akka-docs/src/main/paradox/scala/additional/deploy.md]: Looked like an attractive image but let me know if I should remove it.

File: akka-docs/src/main/paradox/scala/additional/deploy.md [hunk 3]
```diff
@@ -5,7 +5,16 @@ Akka can be used in different ways:
...
  * As an application packaged with [sbt-native-packager](https://github.com/sbt/sbt-native-packager)
- * As an application packaged and deployed using [Lightbend ConductR](http://www.lightbend.com/products/conductr).
+ 
+## Lightbend ConductR
+ 
+An Akka application can also be packaged and deployed using [Lightbend ConductR](https://conductr.lightbend.com/docs/2.2.x/DevQuickStart).
+
+ConductR is a solution for managing Lightbend Reactive Platform applications across a cluster of machines.
+It is reactive from the ground up thus enabling operations to provide the resiliency required to unleash the full benefits 
+of the Lightbend Reactive Platform in their organization.
```
Comment (richard-imaoka) [inline akka-docs/src/main/paradox/scala/additional/deploy.md]: Just copied and pasted the first paragraph from https://conductr.lightbend.com/docs/2.2.x/DevQuickStart

**Comment by richard-imaoka** [comment]
OK, removed the unnecessary image, and renamed from ConductR to Lightbend Enterprise Suite. (using git amend commit for this small change.)

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by ktoso** [comment]
Thanks @richard-imaoka ! :)

**Pull request merged**